### ISSUE TITLE
package_name could not be resolved with new Amazon Linux 2017.03

### ISSUE THREAD

**Issue by christiancsmueller** [chef-cookbooks/httpd#110]
### Cookbook version
0.4.5

### Chef-client version
all

### Platform Details
Amazon Linux 2017.03

### Scenario:
Install HTTPD

### Steps to Reproduce:
Install HTTPD

### Expected Result:
HTTPD installed

### Actual Result:
Error Message: You must supply a name when declaring a package resource

package_name has no standard value and for AmazonLinux 2017.03 no package name could be found in the local map.
-> Cookbook failing

**Comment by eric-tucker** [comment]
Is this the same issue the MySQL cookbook went through?  A big giant hash map of version to package?

**Comment by eric-tucker** [comment]
Encountering the same issue.  Need a fix soon please.

**Comment by tas50** [comment]
Fixed in master

**Comment by eric-tucker** [comment]
Unfortunately my fix under #112 just opened the door to the next issue - there is no template for 2017.03.

```
      ================================================================================
      Error executing action `create` on resource 'template[/etc/init.d/httpd-default]'
      ================================================================================
      
      Chef::Exceptions::FileNotFound
      ------------------------------
      Cookbook 'httpd' (0.4.5) does not contain a file at any of these locations:
        templates/amazon-2017.03/2.4/sysvinit/el-2017/httpd.erb
        templates/amazon/2.4/sysvinit/el-2017/httpd.erb
        templates/default/2.4/sysvinit/el-2017/httpd.erb
        templates/2.4/sysvinit/el-2017/httpd.erb
```
